es6-classes/example-class.js

```diff
@@ -0,0 +1,18 @@
+class ExampleClass {
+  constructor() {
+    this.something = undefined;
+  }
```

Comment:
You can leave out the constructor if you don't need to add any properties to `this`.